android/src/main/java/com/instabug/instabugflutter/ArgsRegistry.java

```diff
@@ -160,4 +162,40 @@ static void registerLocaleArgs(Map<String, Object> args) {
         args.put("Locale.Slovak", new Locale(SLOVAK.getCode(), SLOVAK.getCountry()));
         args.put("Locale.Czech", new Locale(CZECH.getCode(), CZECH.getCountry()));
     }
+
+    @VisibleForTesting
+    static void registerCustomTextPlaceHolderKeysArgs(Map<String, Object> args) {
```

Comment:
We missed a few place holders:
1. `VIDEO_PLAYER_TITLE`
2. `FEATURES_REQUEST`
3. `FEATURES_REQUEST_ADD_FEATURE_TOAST`
4. `FEATURES_REQUEST_ADD_FEATURE_THANKS_MESSAGE`
5. `SURVEYS_WELCOME_SCREEN_TITLE`
6. `SURVEYS_WELCOME_SCREEN_SUBTITLE`
7. `SURVEYS_WELCOME_SCREEN_BUTTON`
8. `REQUEST_FEATURE`